packages/flow-core/src/components/f-select/f-select.ts

```diff
@@ -844,13 +855,14 @@ export class FSelect extends FRoot {
 						${this.type === "single"
 							? (this.selectedOptions as FSelectOptionsProp).map(
 									option =>
-										html`<f-div padding="none" ellipsis=${true}
+										html`<f-div padding="none" .style=${this.singleSelectionStyle}
```

Comment:
remove `.` since it is not lit property , for native html properties we don't need a dot